### PR TITLE
Clarify packstream path indices

### PR DESCRIPTION
**Pull request by robsdedude**

### CODE REVIEW

**Review by AndyHeap-NeoTech — APPROVED** [review]
LGTM

File: packstream/packstream-specification-1.md [hunk 1]
```diff
@@ -676,12 +676,33 @@ B3 72
...
+
+Example (simplified notation for `<Node>` and `<UnboundRelationship>`):
+```
+Path::Structure(
+    nodes: [Node::Structure(42, ...), Node::Structure(69, ...), Node::Structure(1, ...)],
+    rels: [UnboundRelationship::Structure(1000, ...), UnboundRelationship::Structure(1001, ...)],
+    indices: [1, 1, 1, 0, -2, 2],
+```
+
+This represents the path `(42)-[id:1000]->(69)-[id:1000]->(42)<-[1001]-(1)`
```
Comment (robsdedude) [inline packstream/packstream-specification-1.md]:
```suggestion
This represents the path `(42)-[1000]->(69)-[1000]->(42)<-[1001]-(1)`, where `(n)` denotes a node with id `n` and `[n]` a relationship with id `n` (`->` or `<-` denotes the direction of each relationship).
```

**Comment by martin-neotech** [comment]
@AlexicaWright this PR has not been reflected in the https://github.com/neo4j-drivers/docs-bolt/blob/main/modules/ROOT/pages/packstream/index.adoc

@AndyHeap-NeoTech now we have two repositories with the "same" content.

**Comment by AlexicaWright** [comment]
In the other repo: https://github.com/neo4j-drivers/docs-bolt/pull/15

**Pull request merged**